great_expectations_cloud/agent/config.py

```diff
@@ -9,6 +9,7 @@ class GxAgentEnvVars(BaseSettings):
     gx_cloud_base_url: AnyUrl = CLOUD_DEFAULT_BASE_URL
     gx_cloud_organization_id: str
     gx_cloud_access_token: str
+    enable_progress_bars: bool = False
```

Comment:
Let's make this default true, and then set it to false in the agents we are running. This way when someone is testing out an agent the progress bars are on which is the current state.